pkg/testrunner/result/output.go

```diff
@@ -268,3 +276,23 @@ func getOSConfig(tmClient kubernetes.Interface, namespace, minioEndpoint string,
 		BucketName: minioConfig.Data["objectstore.bucketName"],
 	}, nil
 }
+
+// buildKibanaWorkflowURL construct a valid kibana url from the given endpoint and workflow
+// typical endpoint: https://kibana.ingress.<my-cluster-domain>
+// typical kibana url: https://kibana.ingress.<my-cluster-domain>/app/kibana#/...
+func buildKibanaWorkflowURL(endpointPath string, namespace string, workflow string) string {
+	const IndexPatternID = "d1134060-5189-11e9-b7dc-3fcc67c9fd25"
```

Comment:
nit: lowercase variable name `indexPatternID`

Reply:
you're right, changed it (confused it with the uppercase requirement for the respective struct for json marshalling)